stock_picking_auto_create_lot_biochousa/models/product.py

```diff
@@ -6,4 +6,10 @@
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
-    exception_eco_partner_ids = fields.Many2many('res.partner', string="Exception Eco Productor")
\ No newline at end of file
+    exception_eco_partner_ids = fields.Many2many('res.partner', string="Exception Eco Productor")
+
+
+class PartnerTemplate(models.Model):
```

Comment:
Two changes requested:
* A more suitable name for this class should be `ResPartner`.
* The whole class should be placed in a new file (named `res_partner.py`)